gooddata-sdk/tests/catalog/test_catalog_organization.py

```diff
@@ -19,5 +19,45 @@ def test_get_organization(test_config):
     sdk = GoodDataSdk.create(host_=test_config["host"], token_=test_config["token"])
     organization = sdk.catalog_organization.get_organization()
     assert organization.id == "default"
-    assert organization.name == "Default Organization"
-    assert organization.hostname == "localhost"
+    assert organization.attributes.name == "Default Organization"
+    assert organization.attributes.hostname == "localhost"
+
+
+@gd_vcr.use_cassette(str(_fixtures_dir / "update_oidc_settings.json"))
+def test_update_oidc_settings(test_config):
+    sdk = GoodDataSdk.create(host_=test_config["host"], token_=test_config["token"])
+    default_organization_id = "default"
+    default_organization_name = "Default Organization"
+    default_organization_hostname = "localhost"
+
+    organization = sdk.catalog_organization.get_organization()
+    assert organization.id == default_organization_id
+    assert organization.attributes.name == default_organization_name
+    assert organization.attributes.hostname == default_organization_hostname
+
+    oauth_issuer_location = "test.com"
+    oauth_client_id = "123456"
+    oauth_client_secret = "password"
+
+    sdk.catalog_organization.update_oidc_parameters(
+        oauth_issuer_location=oauth_issuer_location,
+        oauth_client_id=oauth_client_id,
+        oauth_client_secret=oauth_client_secret,
+    )
+
+    updated_organization = sdk.catalog_organization.get_organization()
+    assert organization.id == default_organization_id
+    assert organization.attributes.name == default_organization_name
+    assert organization.attributes.hostname == default_organization_hostname
+    assert updated_organization.attributes.oauth_issuer_location == oauth_issuer_location
+    assert updated_organization.attributes.oauth_client_id == oauth_client_id
+
+    sdk.catalog_organization.update_oidc_parameters(
```

Comment:
I would call the previous code with "try except" to ensure that this revert happens every time.

Reply:
Done.